util/mathutil/mathutil_js.go

```diff
@@ -0,0 +1,102 @@
+package mathutil
+
+const MaxUint = ^uint(0) 
+const MinUint = 0 
+const MaxInt = int(MaxUint >> 1) 
+const MinInt = -MaxInt - 1
+
+func MaxUint64(x, y uint64) uint64 {
```

Comment:
please add comments for the export functions and vars

Reply:
Ok, thanks for reminding